### ISSUE TITLE
Filemanager's default interfers with switch's default.

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#28]
Hi Nick,

````swift
switch x {
    case a:
        try FileManager.default.removeItem(atPath: z)
    default:
        break
}
````
fails to be parsed (v0.9.3). It is confused by the `default` of `Filemanager` (as Github syntax highlighting!).

**Comment by nicklockwood** [comment]
Interesting. This wouldn't even compile under Swift 2.2,  but it seems Swift 3 is a lot more relaxed about keywords.

Thanks for the report - I'll put up a fix asap.

**Comment by RabugenTom** [comment]
`NSFileManager.defaultManager()` as been renamed `Filemanager.default` with Swift 3. The same with `NotificationCenter.default`.

**Comment by nicklockwood** [comment]
So I gathered. Anyway, this is fixed in 1.9.4. Thanks again.
